app/components/Location.tsx

```diff
@@ -0,0 +1,23 @@
+import { Box } from "@chakra-ui/react";
+
+const content = [
+  "Our family-owned CNC production company is located in Duvall, Washington, just 30 miles northeast of the Seattle metropolitan area. Clients of Kenworthy Machine appreciate our local accessibility as well as our statewide service and delivery.",
+];
+
+const Location = () => {
+  return (
+    <Box
+      as="p"
+      bg="brand.backgroundMed"
+      fontSize={{ base: "2rem", xl: "3rem" }}
```

Comment:
This looks quite large on my screen. I wonder if we should start using the wireframe/Figma measurements/sizes for ratios only and use our own judgment to set sizes? This is also a issue in the values section I just created. Also, should we use the Chakra theme font sizes here?
I tried this: `fontSize={{ base: "3xl", xl: "4xl" }}` and it looks pretty good
<img width="1436" alt="Screenshot 2024-01-31 at 2 43 55 PM" src="https://github.com/wavecrestweb/kenworthy-machine/assets/115492619/c0d0f71b-85f2-46de-807a-1d5d45ff01c4">

Reply:
Yes, I agree that sizing needs to be scaled down as I mentioned in the PR description. And yes, your suggested sizes look great, thank you! I'll also adjust line height and padding based on font size changes. 

As for using Chakra theme font sizes, I don't mind using them; though I find it easier to use rems when translating wireframes' pixels (dividing px by 16). Chakra theme font sizes do not always match what I need in terms of rems (I use [this](https://chakra-ui.com/docs/styled-system/theme#typography) to check the sizing in rems for Chakra font sizes), so I just use rems instead. However, since we decided to completely overhaul sizing, I guess I can switch ChakraUI default theme sizing for consistency if that's what you all prefer.

Reply:
I feel like we have been moving in the direction of using the Chakra custom theme as our source of truth for our design system specifications. So, maybe we should customize all of the default font sizes, as in:

fontSizes: {
	...
	"xl": "1.25rem",
    "2xl": "1.5rem",
    "3xl": "2rem",
	...
}

That might be an overall DRYer and more Chakra-esque approach to regulating font sizes for the entire app.

Reply:
@BrettEastman I agree, that this would be optimal to override sizing in ChakraUI theme. But Archaa and I had a brief discussion in slack where we leaned towards setting up a doc with conversions for font-sizes/spacing/etc to keep resizing consistent (I'll tag you). That's what I'm adding to this PR. We can discuss it further when you're back and maybe use this doc as a starting point to customize ChakraUI theme.

Reply:
I reduced the font-size even further and adjusted line-height as well. You can refer to the doc that I've added to see the conversions. Let me know if that looks good or needs to be adjusted further

Reply:
The doc looks great. Thank you!